simcon_project/templates/conversation/conversation_end.html

```diff
@@ -30,6 +30,18 @@ <h3 style="text-align:right;">{{ request.user }}</h3>
                     <input type="text" name="{{ response.id }}" id="{{ response.id }}"><br>
                 </div>
             {% endfor %}
+
+            Rate your performance:<br>
+            <input type="radio" name="{{ ct_response.self_rating }}" id="{{ ct_response.self_rating }}" value=1>
+            <label for="{{ ct_response.self_rating }}">1 - Very Unsatisfied</label><br>
+            <input type="radio" name="{{ ct_response.self_rating }}" id="{{ ct_response.self_rating }}" value=2>
+            <label for="{{ ct_response.self_rating }}">2 - Unsatisfied</label><br>
+            <input type="radio" name="{{ ct_response.self_rating }}" id="{{ ct_response.self_rating }}" value=3>
+            <label for="{{ ct_response.self_rating }}">3 - Somewhat Satisfied</label><br>
+            <input type="radio" name="{{ ct_response.self_rating }}" id="{{ ct_response.self_rating }}" value=4>
+            <label for="{{ ct_response.self_rating }}">4 - Satisfied</label><br>
+            <input type="radio" name="{{ ct_response.self_rating }}" id="{{ ct_response.self_rating }}" value=5>
+            <label for="{{ ct_response.self_rating }}">5 - Very Satisfied</label><br>
```

Comment:
Are we not holding onto these descriptions somewhere? As in would we want to use these same descriptions in other views or just here in this html?

Reply:
I could use these on the student home modal too. Adding a tuple to the model seems like an easy way to get the descriptions anywhere

Reply:
I dunno. I feel like it's just a descriptor that doesn't matter too much